packages/block-library/src/latest-posts/block.json

```diff
@@ -12,6 +12,12 @@
 		"categories": {
 			"type": "array"
 		},
+		"users": {
+			"type": "array"
+		},
+		"items": {
```

Comment:
What do we need this items for? It seems that you needed it for the array type, but it should be nested into users. Now that I're recommented switching to number it is not needed at all.

Reply:
Items was an attempt to resolve the issue of the error: "Notice: Undefined index: items in /app/public/wp-includes/rest-api.php on line 1245, but as you mentioned. It's not necessary and I've removed it.

I've also switched the type to number on users and is working as expected.